.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,35 +38,27 @@ See ticket description.
 
 ---
 
-## Added tests?
-- [ ] 👍 yes
-- [ ] 🙅 no, because they aren't needed
-- [ ] 🙋 no, because I need help
----
 
-## Added to documentation?
-- [ ] 📜 README.md
-- [ ] 📓 Confluence
-- [ ] 🙅 no documentation needed
-
-## Added required labels?
+## 🔴 IMPORTANT: Add required labels
 
 At least one of each type of PR label should be added for checks to pass.
 
 - `notes:*`
 - `version:*`
 
-Patch versions are code-only changes, with no data updates.
-Minor versions are data changes, such as updates to the measures-data.json files.
-Major versions are annual and tied to a PY's data becoming avaliable, determined by QPPA's PO.
-
-<!--
-Eg. If this PR contains a regular measures update, add the following labels:
-- notes:measure-update
-- version:minor
--->
-
+**Patch** versions are code-only changes, with no data updates.
+**Minor** versions are data changes, such as updates to the measures-data.json files.
+**Major** versions are annual and tied to a PY's data becoming avaliable, determined by QPPA's PO.
```

Comment:
Moving this higher in the template to increase its visibility to new contributors.